pkg/config/config.go

```diff
@@ -1139,6 +1140,16 @@ type Config struct {
 	// Default: none
 	// Public: Yes
 	NtpMetrics NtpConfig `yaml:"ntp_metrics" envconfig:"ntp_metrics"`
+
+	// AgentTempDir is the directory where the agent stores temporary files (i.e. fb config, discovery...)
+	// It will be DELETED on every afent restart only if it matches default value
```

Comment:
it looks like a typo

Reply:
nice 👁️  fixed in 4e88b3d4a307d077ee012c3b2863620b51112d7a